code/datums/wounds/burns.dm

```diff
@@ -164,18 +165,31 @@
 	. = ..()
 	. += "<div class='ml-3'>"
 
+	. += "Estimated duration of treatment while the healing rate is active: [span_deadsay("[flesh_damage]s")]\n"
+	if(flesh_healing > 0)
+		. += "Positive signs of healing in the flesh damage, rate: [span_green("[flesh_healing]%")]\n"
+	else
+		. += "Flesh damage is not exhibiting signs of healing, rate: [span_brass("[flesh_healing]%")]\n"
```

Comment:
```suggestion
	. += "Wound bed damage: [span_deadsay("[flesh_damage]")]\n"
	if(flesh_healing > flesh_damage)
		. += "Currently applied treatment will fully address present flesh damage\n"
```

Reply:
This should also be moved under line 176, I believe, since the related if check already goes over if the wound has been effectively healed